pkg/cmd/task/start.go

```diff
@@ -18,12 +18,17 @@ import (
 	"context"
 	"encoding/json"
 	"errors"
-	"fmt"
-	"net/http"
 	"os"
 	"strings"
 	"time"
 
+	"github.com/google/go-containerregistry/pkg/name"
```

Comment:
not goimported

Reply:
@williamlfish it should "auto-install" it in the `.bin` folder on the `make lint` commands (or whatever the target name is). At least that's what it's supposed to do 😛

Reply:
@vdemeester  I updated `make lint` to do that with the latest commit. I'm not sure what version or what binary `yamllint` is supposed to be, but we should have that auto install too, for now i just moved to to the end so it fails there if someone does not have it installed 🤷‍♀️